index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
```

Comment:
Great job with implementing the UI. However, as of now, it not matching the Figma template of your choosing. For this Module, you will have to strictly stick to the Figma template that you are choosing. Consider giving it more padding on top of the header.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/212340802-e11d541c-55e9-42d9-b242-f2a9405820b2.png) | ![image](https://user-images.githubusercontent.com/33268927/212340704-5353bcd1-8e3e-4538-be9e-3d5b3f0ce9da.png)|